SRS.md

```diff
@@ -20,16 +20,13 @@ The best way to decrease expenses and increase profit in the restaurant business
 The proposed system will increase order accuracy and decrease the time between the customer sitting down and receiving their food. It will also decrease the time the customer has to wait to pay for their order. This should increase the overall satisfaction of the customer, promoting return business.
 
 ##### 1.2 - Scope
-To start, we are going to focus only on the customers, the servers, and the kitchen. By providing a middle-system, we will be able to speed up the 
+To start, we are going to focus only on the customers, the servers, and the kitchen. By acting as an intermediate part, we will be able to speed up the 
 process of ordering food, getting the order to the kitchen, and getting completed orders to tables. Not only will this system benefit the customer, 
-but servers will now be able to manage their tables and orders more efficiently.
+but servers will now be able to manage their tables and orders more efficiently. The app will be used for several business promotion and customer can quickly give their recommendations to their dear ones.
```

Comment:
Not sure if we need this sentence. We don't really mention this anywhere else.

Reply:
ok i will remove it them